Improve breaking changes info for CRA 3.0.0 (#7086)
facebook/create-react-app: CHANGELOG.md

We've removed the `--no-watch` flag from the `start` script in favor of Jest's own `--watchAll=false`.

### New structure in `asset-manifest.json`

> All paths have been moved inside of the `files` key inside `asset-manifest.json`.

Reviewer: ```suggestion
All asset paths have been moved under the `files` key in `asset-manifest.json`.
```

- `react-scripts`

>   - [#6845](https://github.com/facebook/create-react-app/pull/6845) Change CRA version in `react-scripts` eject warning. ([@lffg](https://github.com/lffg))

Reviewer: This should still be listed here.

Author: Done.
